src/main/java/com/thirdcc/webapp/web/rest/EventAttendeeResource.java

```diff
@@ -114,6 +116,21 @@ public EventAttendeeResource(EventAttendeeService eventAttendeeService) {
         return ResponseUtil.wrapOrNotFound(eventAttendeeDTO);
     }
 
+    /**
+     * {@code GET  /event-attendees/event/{eventId}} : GET the eventAttendee in "eventId" event.
+     *
+     * @param eventId the id of the event to retrieve list of eventAttendeeDTO.
+     * @return the {@link ResponseEntity} with status {@code 200 (OK) or 400 (BadRequest)}.
+     */
+    @GetMapping("/event-attendees/event/{eventId}")
+    @PreAuthorize("hasRole(\"" + AuthoritiesConstants.ADMIN + "\")")
```

Comment:
@junzai97  as refer to [Decision Before](https://github.com/Five-Fishes/Club-Management-React-Client/issues/10)
Event Crew and above Authority should can have access to view event attendee

Reply:
resolved in df95cf3de79d1327249d112fd9904cc1fe142879